apps/timetable/admin/app/js/index.js

```diff
@@ -113,7 +113,7 @@ define(['gh.core', 'gh.constants', 'jquery-autosuggest', 'validator'], function(
      * @private
      */
     var renderConfig = function(config) {
-        gh.utils.renderTemplate('tenant-admin-config', {
+        gh.utils.renderTemplate('admin-app-config', {
```

Comment:
Same here. Basically all files that have a filename that differs from the one we use to pass into the template functionality need to be updated.